vyper/parser/parser.py

```diff
@@ -57,13 +61,6 @@
 )
 
 
-# Header code
-INITIALIZER_LIST = ['seq', ['mstore', 28, ['calldataload', 0]]]
```

Comment:
Can you include the deletion of this constant in a separate commit with a message that explains why it was removed?  I had to do a bit of poking around to figure out why this was taken out.  It looks like it just wasn't used anymore?

Reply:
yeah i discovered it was dead code. i can split it into a separate commit

Reply:
Actually I remember why I put it in the same commit, it's because it had duplicated code with STORE_CALLDATA and I didn't want the commit to change one of the `calldataload` without changing the other one.

Reply:
I added a note to the commit message to clarify.